chia/util/condition_tools.py

```diff
@@ -74,35 +76,35 @@ def pkm_pairs(npc_list: List[NPC], additional_data: bytes) -> Tuple[List[bytes48
                     assert len(cwa.vars) == 2
                     assert len(cwa.vars[0]) == 48 and len(cwa.vars[1]) <= 1024
                     assert cwa.vars[0] is not None and cwa.vars[1] is not None
-                    ret[0].append(bytes48(cwa.vars[0]))
+                    ret[0].append(G1Element.from_bytes(cwa.vars[0]))
```

Comment:
This hurts performance since we need to perform the subgroup checks

Reply:
This was also a bad merge. I'll change this back.